pkg/detectors/onfido/onfido.go

```diff
@@ -0,0 +1,91 @@
+package onfido
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"strings"
+	"time"
+	regexp "github.com/wasilibs/go-re2"
+
+	"github.com/trufflesecurity/trufflehog/v3/pkg/common"
+	"github.com/trufflesecurity/trufflehog/v3/pkg/detectors"
+	"github.com/trufflesecurity/trufflehog/v3/pkg/pb/detectorspb"
+)
+
+type Scanner struct {
+	client *http.Client
+}
+
+// Ensure the Scanner satisfies the interface at compile time.
+var _ detectors.Detector = (*Scanner)(nil)
```

Comment:
Unfortunately, With the recent changes, There is another func `Description` introduce in `Scanner` interface. Code will not compile without that implementation.